client/my-sites/stats/stats-module/connected-list.js

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
I suspect if you rebase you'll see a handful of [`jsx-classname-namespace`](https://github.com/Automattic/eslint-plugin-wpcalypso/blob/master/docs/rules/jsx-classname-namespace.md) warnings in this file.